packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -95,12 +102,22 @@ async function runPipelineLoadStep(
           // if source maps disabled, don’t return any
           if (!sourceMaps) result[ext].map = undefined;
         });
+        if (devMessageBus && isDev) {
+          devMessageBus.emit('SUCCESS', {id: step.name});
```

Comment:
already talked through this in person, but this is the bit that isn't really compatible with our current idea of a whole plugin b eing SUCCESS or FAIL.

example 1:
- `App.svelte` loads in the dev server, has an error, fails
- `Icon.svelte` also loads, succeeds, "SUCCESS" overwrites "ERROR"
- End result is no error shown

example 2:
- `App.svelte` loads in the dev server, has an error, fails
- "Svelte" plugin is now in the "error" state
- What does it mean to go back to the "success" state? Do we need to track which files failed, and then only go to "SUCCESS" state when that file succeeds? what if the file is deleted? etc etc

Reply:
Right. I thought of that convo while doing this, and I agree it’s a bit hacky. But one thing I realized in playing with it was it felt “broken” in a sense to show the `READY` status if the dev server was running but there was an error and nothing was showing. It felt a lot better to either show `READY` or `ERROR` based on the state, but to your point this isn’t a perfect implementation.

Maybe we change that `READY` status to `RUNNING`? Or something else? Or maybe we just take the green color away? Open to a solution where an error is visible and it doesn’t feel like the dev server is “lying” to us telling us everything is good when it’s not.